### DISCUSSION TITLE
Automating my home setup: turning on the lights when the camera is in use

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#8] [Comments]
# Automating my home setup: turning on the lights when the camera is in use

Automating my home setup: turning on the lights when the camera is in use

https://devopsjournal.io/blog/2022/01/17/Automating-my-presence

**Comment by loryanstrant** [comment]
Good to see a fellow Home Assistant afficionado integrating it with M365!
You might find this post I did of interest to toggle workflows based on your Microsoft Teams presence without requiring as much happening on your desktop: https://www.loryanstrant.com/2022/03/04/microsoft-teams-home-assistant-graph-take-your-presence-to-the-next-level/

**Reply by rajbos** [reply]
Thanks! And indeed, that would work as well ;-). Thing is: I use so many different tools across all of my customers (and 4 different agenda's at the moment, 3 different tenants, don't ask)! Teams does not know have of the time when I am available or not :-D. Therefor I look at the things that always know if I am in a call or not: the camera's!
